src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <header class="header"></header>
+    <main class="page forms-page">
+      <div class="forms-page__container">
+        <form
+        data-qa="big"
+        action="#"
+        method="post"
+        class="forms-page__form-big"
+        >
```

Comment:
```suggestion
        <form
          data-qa="big"
          action="#"
          method="post"
          class="forms-page__form-big"
        >
```
fix it everywhere - [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 5
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/d834fad1-06ae-4d5f-b758-febcd21f7f8e)